src/puppetlabs/trapperkeeper/services/scheduler/job.clj

```diff
@@ -1,8 +1,8 @@
 (ns puppetlabs.trapperkeeper.services.scheduler.job
   (:gen-class
    :name puppetlabs.trapperkeeper.services.scheduler.job
-   :state "state"
-   :init "init"
+   :state state
+   :init init
```

Comment:
Why this? I see examples using both forms but none that explain the difference...

Reply:
This cryptically fails the ns spec when upgrading clj-parent. I had no idea what spec was trying to tell me and in the examples for ns they used symbols rather than strings.... I tried that it and it stopped yelling at me....